peachjam/resources.py

```diff
@@ -146,31 +163,51 @@ def before_import_row(self, row, **kwargs):
                 defaults={"name": row["court_obj"]["name"]},
             )
 
+        source_files = row["source_url"].split("|")
+        docx = re.compile(r".docx?$")
+        pdf = re.compile(r".pdf$")
+        for file in source_files:
+            match_docx = docx.search(file)
+            match_pdf = pdf.search(file)
+            if match_docx:
+                row["source_url"] = file
+            elif match_pdf:
+                row["source_url"] = file
+

```

Comment:
this doesn't make any sense -- you're setting `source_url` to the same thing each time.

Reply:
I have added  a break in the loop to terminate when it encounters a .docx file. Otherwise it will accept a pdf file if it doesn't find a docx.